scss/core/core.scss

```diff
@@ -5,7 +5,7 @@
 @import "~bootstrap/scss/reboot";
 @import "typography";
 @import "grid";
-@import "~react-loading-skeleton/dist/skeleton";
+@import "~react-loading-skeleton/dist/skeleton.css";
```

Comment:
For some reason, the dev server won't start for me without this `.css` file extensions after upgrading `react-loading-skeleton`.